optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/ScoreDirectorType.java

```diff
@@ -0,0 +1,10 @@
+package org.optaplanner.core.impl.score.director;
+
+public enum ScoreDirectorType {
+
+    EASY,
+    INCREMENTAL,
+    DROOLS,
+    CONSTRAINT_STREAMS
```

Comment:
Order should be:
easy, constraint_streams, drools, incremental IIRC.
Definitely CS before DRL these days.

Reply:
Can we rename DROOLS to DRL for clarity?